build.gradle

```diff
@@ -27,7 +27,7 @@ buildscript {
         common_utils_version = System.getProperty("common_utils.version", '3.0.0.0-SNAPSHOT')
 
         kafka_version  = '3.7.1'
-        apache_cxf_version = '4.0.4'
+        apache_cxf_version = '4.0.5'
```

Comment:
nit: this might not be needed on newer versions after https://github.com/opensearch-project/security/pull/3421

Reply:
Right, looks like cxf can be removed on the head of main and 2.x. The only references remaining are in [CxfTestTools](https://github.com/opensearch-project/security/blob/main/src/test/java/com/amazon/dlic/auth/http/jwt/keybyoidc/CxfTestTools.java), but it appears that class is not used. Should we merge this PR and trigger that automatic backport process or specifically target 1.3 for this change?

Reply:
I plan to close this PR and open a targeted one based on 1.3.

I opened a separate PR to remove CXF from main and 2.x since there's no longer any references that are used: https://github.com/opensearch-project/security/pull/4580

Reply:
Hi @cwperks , are you still planning to open a PR targeting 1.3 with this change? 
If not I can take over. Just don't want to step on your toes.